core-schema/src/main/resources/sql-schema/V8__Mod_clans.sql

```diff
@@ -86,3 +87,29 @@ CREATE FUNCTION clans_add_enemy
     INSERT INTO clans_clan_enemies (clan_id, enemy_id) VALUES (clan_identifier, enemy_identifier);
     RETURN 0;
   END;
+
+CREATE FUNCTION clans_create_new (name VARCHAR(32), leader_id INT) RETURNS TINYINT MODIFIES SQL DATA
```

Comment:
Since this function is unused, do we have to keep it around? We can always add it back from version control if necessary (also a good reason why to keep old branches around)

Reply:
we will delete it